release_tester/mixed_download_upgrade_test.py

```diff
@@ -218,8 +229,10 @@ def upgrade_package_test(
 
     # STEP 4: Run other test suites
     if run_test_suites:
-        enterprise_packages_are_present = "EE" in editions or "EP" in editions
-        community_packages_are_present = "C" in editions
+        enterprise_packages_are_present = (
```

Comment:
shoulb we make the two a function near EXECUTION_PLAN so we can DRY our code?

Reply:
Now the flags are being set when packages are being downloaded. That is more correct.